### ISSUE TITLE
Docker build fails - need aarch64-unknown-linux-gnu

### ISSUE THREAD

**Issue by sanderdms** [dusk-network/rust#9]
Currently the Docker Build is failing because `setup-compiler.sh` cannot complete unzipping aarch64-unknown-linux-gnu.zip from this repository.
https://github.com/dusk-network/rusk/blob/3ad0e521e21e9b949772bc65b37f663891dafb7d/scripts/setup-compiler.sh#L37

Isolated test Dockerfile

`docker build . --no-cache`

    FROM rust:latest
     
    RUN echo "Downloading compiler version v0.2.0" 
    RUN curl -L https://github.com/dusk-network/rust/releases/download/v0.2.0/aarch64-unknown-linux-gnu.zip -o "download.zip" 
    RUN unzip download.zip

```
Extracting compiler...
End-of-central-directory signature not found. Either this file is not
a zipfile, or it constitutes one disk of a multi-part archive. In the
latter case the central directory and zipfile comment will be found on
the last disk(s) of this archive.
unzip: cannot find zipfile directory in one of /opt/rusk/target/dusk/compiler/v0.2.0/duskc-aarch64-unknown-linux-gnu.zip or
/opt/rusk/target/dusk/compiler/v0.2.0/duskc-aarch64-unknown-linux-gnu.zip.zip, and cannot find /opt/rusk/target/dusk/compiler/v0.2.0/duskc-aarch64-unknown-linux-gnu.zip.ZIP, period.
make: *** [Makefile:62: setup-compiler] Error 9
```

`docker --version` Docker version 25.0.3, build 4debf41
`uname -a` :Darwin Kernel Version 23.3.0: Wed Dec 20 21:30:27 PST 2023; root:xnu-10002.81.5~7/RELEASE_ARM64_T8103 arm64 (M1 MacBook Air)

**Comment by lklimek** [comment]
Similar error happens on "generic" arm64 + debian

**Comment by HDauven** [comment]
@lklimek @sanderdms this issue is resolved by PR #13. I added an ARM/Linux version of the compiler to our [0.2.0 release](https://github.com/dusk-network/rust/releases/tag/v0.2.0).
